[ZT] Gateway security categories (#6582)
cloudflare/cloudflare-docs: content/cloudflare-one/policies/filtering/dns-policies/_index.md

### Security Categories

Use this selector to block traffic directed to specific [security categories](/cloudflare-one/policies/filtering/domain-categories/#security-categories).

| UI name | API example |
| -- | -- |
| Security Categories | `any(dns.security_category[*] in {1})` |

> When creating a DNS block policy for security categories, you can optionally enable **Filter traffic by resolved IP category** in the policy settings. When enabled, Gateway will block queries based on their resolved IP address instead of domain name. This setting may increase the number of false positives because domains in the blocked category can share IP addresses with legitimate domains.

Author: ```suggestion
When creating a DNS block policy for security categories, you can optionally enable **Filter traffic by resolved IP category** in the policy settings. When enabled, Gateway will block queries based on the resolved IP address in addition to the domain name. This setting may increase the number of false positives because domains in the blocked category can share IP addresses with legitimate domains.
```
